src/include/duckdb/storage/buffer/buffer_pool.hpp

```diff
@@ -85,10 +88,35 @@ class BufferPool {
 	void IncrementDeadNodes(FileBufferType type);
 
 protected:
+	struct MemoryUsageCounters {
+		//! The maximum difference between memory statistics and actual usage is 2MB (64 * 32k)
+		static constexpr size_t kCacheCnts = 64;
+		static constexpr size_t kCacheThreshold = 32 << 10;
```

Comment:
nit: this doesn't match case style used elsewhere, can we change the names here to e.g. `TAG_CACHE_COUNT`/`TAG_CACHE_THRESHOLD`?